src/client/Microsoft.Identity.Client/MsalException.cs

```diff
@@ -39,9 +45,11 @@ public MsalException()
         /// The error code returned by the service or generated by the client. This is the code you can rely on
         /// for exception handling.
         /// </param>
-        public MsalException(string errorCode)
+        /// <param name="isRetryable">The boolean to indicate whether to perform retry operation for the exception.</param>
+        public MsalException(string errorCode, bool isRetryable = false)
```

Comment:
>public MsalException(string errorCode, bool isRetryable = false) [](http://example.com/codeflow?start=8&length=64)

are these constructs not (binary) breaking changes? Would there be a point in keeping the existing methods and extending with the additional ones? 
(same comment multiple places)

Reply:
I have reverted the change to the constructors. Added the variable as internal set.